core/templates/dev/head/components/response_header_directive.html

```diff
@@ -70,7 +77,7 @@
         ng-click="navigateToState(getOutcome().dest)" class="oppia-nested-link">
     <[getOutcome().dest]>
   </span>
-  <span ng-if="getOutcome() && !isOutcomeLooping() && isCreatingNewState()">
+  <span class="oppia-nested-link" ng-if="getOutcome() && !isOutcomeLooping() && isCreatingNewState()">
```

Comment:
Why this change?

Reply:
@nithusha21 
It fixed this behaviour. Earlier, the text that denoted new card name overlapped with the close icon.
### Before:
![before](https://user-images.githubusercontent.com/32777813/50449652-a4e3c380-094e-11e9-8e2f-01695430e20b.gif)

### After:
![after](https://user-images.githubusercontent.com/32777813/50449680-dc527000-094e-11e9-96ba-8344c74eab6e.gif)